scripts/commands/checkmission.lua

```diff
@@ -13,7 +13,20 @@ cmdprops =
 
 function onTrigger(player,logId,target)
     
+    local logName;
     logId = tonumber(logId) or _G[logId];
+    if ((type(logId) == "table")) then
+        logName = logId.full_name;
+        logId = logId.mission_log;
+    else
+        if (logId <= 2) then
+            local logNames = {"San d'Oria", "Bastok", "Windurst"}
```

Comment:
@ibm2431 we have a prob here in need of fixing. `San d'Oria` can never be input because the command interpretation would see it as multiple arguments. Better off being `sandoria`, `bastok`, `windurst` and fed through `string.upper()` to kill any case sensitivity. I don't know where else these strings got used exactly, no time to go look.